scripts/check_duplicates.py

```diff
@@ -64,7 +64,7 @@ def create_default_sources():
     return sources
 
 
-def check_duplicates(sources):
+def check_duplicates(sources, os_name, os_version):
```

Comment:
Please use the variable name `os_codename` instead of `os_version` throughout the patch since that is the term used in the REP for this value.